### ISSUE TITLE
I'm considering renaming zVirtualDesktop

### ISSUE THREAD

**Issue by mzomparelli** [mzomparelli/zVirtualDesktop#121]
So I'm looking for name suggestions from you guys. I don't think the current name is bad, it's just a generic naming convention I use for my projects until I come up with a proper name.

**Comment by SolveAllProblemsRightNow** [comment]
focus on having some simple installer

none of  these other things matters

but the best kind of naming eventually is wtf this does

so chrome would be call 'OpenTheWeb' or 'AccessTheWeb' or 'WebOS' or 'OntheWeb' or best is like 'Surf' or maybe 'OneWeb'

win10 should be call 'MicrosoftOS' or 'MicroOS.....' hahahhah

ios is a good name

iphone is a fine name

firefox is a 'cool' name its fine

atom is fine

pycharm is meh

steam is bad

discord is terrible

battle.net is bad

onenote is good

evernote is not that good

forever note is better

everything naming out of linux is always fucking bad

the naming should be wtf this does
 
the naming of other things across anything else is done for other reasons that would not matter here

the naming should eventually be wtf this does
 
anything in the long future, feel free to contact to me (however that works on github... (github is a terrible awful name, git is fine

edge is a good name, but havent ever use it

webtime tracker is a **perfect** name -- https://chrome.google.com/webstore/detail/webtime-tracker/ppaojnbmmaigjmlpjaldnkgnklhicppk

onetab is solid in naming and function -- https://chrome.google.com/webstore/detail/onetab/chphlpgkkbolifaimnlloiipkdnihall

**Comment by mzomparelli** [comment]
zVirtualDesktop says what it does so I should keep as it is then?

As far as simpler install....I'm trying to have a Cholcolatey package built, but that process confuses me. I'm ready for help on that.

**Comment by mzomparelli** [comment]
Someone else made a program like this (not as feature rich) that cost $10. The name of that program...Peach

LOL

**Comment by SolveAllProblemsRightNow** [comment]
im sure there's a chocolaty offical forum that would solve any of that

have no comments that things i haven't use yet and dont understand, but dont see why there's a z in the naming

'spaces' is the best thing, but dunno what this does

spaces is:

* clean

* descriptive enough

* simple

* fresh

* short

maybe a girl made peaches

maybe someone like mario a plenty

maybe it just someone that doesnt know how to name things

maybe the person wanted to be the next apple

i like the name tho, it's 'positive' - like it better than 'pycharm' just slightly

i like peaches, probably, they're 'probably' yummy =)

facebook is likely one of the best naming out there, friendbook is better, but naming was done earlier on when they didnt know wtf it does

**Comment by mzomparelli** [comment]
the z is for my last name and that z is as an awesome letter lol

it makes it unique so that one can search for the exact name.

**Comment by SolveAllProblemsRightNow** [comment]
'unique'

nobody needs to search for it

look at all the (relatively) common words used in namings

'apple maps'

'chrome'

'atom'

'deep learning'

when someone needs to search for a specific thing/product/anything, they'll be able to find it then, no need to worry about, as said none of this matters at this point

things are found in many many many other ways

'unique' is important for functions and many things, but for naming, it's one of the least importnat thigns

should something be 'unique enough' - sure why not, no harm there, and that's why a name like 'apple' isn't good -- but this is criteria is a very low threshold to reach

the list at top shows that

---

'spaces z'

'Spaces Z'

'OneSpace'

'OneDesktop'

'ManyScreens'

'MutiScreens'

'ManySpaces'

'ManyDesktops'

**Comment by nicolasdejong** [comment]
I think it is a good idea to look for a catchier name. Since zVirtualDesktop is an addition to the Windows virtual desktops, the name should probably include that with like a 'plus' (or 'enhancer' but that is a bit long). So how about these:
- VDeskPlus
- WinDeskPlus
- WinVirtPlus
- WindowsVirtualDesktopEnhancer (WVDE)

Hmm. WinDeskPlus is my favorite upto now.

**Comment by mzomparelli** [comment]
Thanks for the great suggestions!

**Comment by mzomparelli** [comment]
VirtualDesktop++

**Comment by nicolasdejong** [comment]
Also a nice one. Just note that Google search is weird with pluses because of their (failed) silly social network

**Comment by sortofworks** [comment]
TL;DR;
Name it to something similar to what people would put in the search engine to solve this problem. 

Longer:
It's an awesome app, and it would be awesome if more people found their way to this site. I found the app by googling, ending up on StackOverflow where someone posted an answer mentioning this app. The answer was one of the last ones, hence only tenacious people that scrolls to the bottom will find it.

My specific search query was "windows 10 move window to other virtual desktop". 

Heaps of search hits takes you to sites that moves windows to second monitors  (WIN+Shift + Arrow). This is not what I wanted.

**Comment by dsp0105** [comment]
I found zvirtualDesktop after lot of effort.

Google search results were hijacked by junk virtual desktop applications. Thats sad since they were coming up first in search and a high quality one like yours didn't come at all.

I wouls day zVirtualDesktop is a good name , may be add Plus to tell it is better (plus seems inane may be catchier than that). 

Please work on getting yours in the google search for virtual desktop, I cant stand seeing those crappy autohotkey ones or dumb ones hogging the results!

**Comment by dsp0105** [comment]
VirtualDesktop++ 

Sounds awesome :)
